app/assets/javascripts/Components/one_time_annotations_table.jsx

```diff
@@ -0,0 +1,244 @@
+import React from 'react';
+import {render} from 'react-dom';
+
+import ReactTable from 'react-table';
+import CreateModifyAnnotationPanel from './Modals/create_modify_annotation_panel_modal';
+
+const INITIAL_ANNOTATION_MODAL_STATE = {
+  show: false,
+  onSubmit: null,
+  title: '',
+  content: '',
+  categoryId: '',
+  isNew: true,
+  changeOneOption: false,
+};
+class OneTimeAnnotationsTable extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      data: [],
+      annotationModal: INITIAL_ANNOTATION_MODAL_STATE,
+      loading: true
+    };
+  }
+
+  componentDidMount()  {
+    this.fetchData();
+  }
+
+  fetchData = () => {
+    $.ajax({
+      url: Routes.uncategorized_annotations_assignment_annotation_categories_path(this.props.assignment_id),
+      dataType: 'json',
+    }).then(res => {
+      this.setState({
+        data: res,
+        loading: false
+      });
+    });
+  };
+
+  refreshAnnotations = () => {
+    $.ajax({
+      url: Routes.uncategorized_annotations_assignment_annotation_categories_path(this.props.assignment_id),
+      dataType: 'json',
+    }).then(res => {
+      this.setState({
+        data: res,
+        loading: false
+      })
+    });
+  };
+
+  editAnnotation = (annot_id, content) => {
+    let onSubmit = (formData) => {
+      return ($.ajax({
+        url: Routes.update_annotation_text_assignment_annotation_categories_path(this.props.assignment_id)+ '?id=' + annot_id,
```

Comment:
The `id` should be passed as `data`. But also I'm not sure why this id is necessary; it looks like the `formData` should contain the `annotation_text_id` already.